src/ts/universes/level_1.ts

```diff
@@ -1,10 +1,28 @@
-import { GoalPlanet, Planet, UniversePlayer } from '../universe';
+import { GoalPlanet, Planet, UniversePlayer, UniverseText } from '../universe';
 import { buildUniverse, UniverseBuilder } from './builder';
 
 export const universe: UniverseBuilder = buildUniverse(() => ({
   objects: [
-    new UniversePlayer({ x: 100, y: 100, orientation: Math.PI / 4 }),
-    new Planet({ x: 100, y: 100, texture: '#f00' }),
-    new GoalPlanet({ x: 300, y: 300, texture: '#f00', goalTexture: '#ff26' }),
+    new UniversePlayer({ x: 400, y: 200, orientation: Math.PI / 4 }),
+    new Planet({ x: 400, y: 200, texture: '#f00' }),
+    new GoalPlanet({ x: 800, y: 300, texture: '#f00', goalTexture: '#ff26' }),
+    new UniverseText({
+      x: 640,
+      y: 450,
+      fontSize: 32,
+      text: 'Welcome to Planet Hoppers!',
+    }),
+    new UniverseText({
+      x: 640,
+      y: 500,
+      fontSize: 24,
+      text: 'Hold the spacebar to charge your jump',
+    }),
+    new UniverseText({
+      x: 640,
+      y: 550,
+      fontSize: 24,
+      text: 'release to get to your destination planet!',
```

Comment:
You could just add a `\n` to get a newline and avoid multiple text objects

Reply:
I tried that and for some reason it just ignored the newline? Any idea why it would do that?

Reply:
Looks like canvas doesn't support it